### DISCUSSION TITLE
Import problem

### DISCUSSION THREAD

**Discussion by rsmpls** [quillforms/quillforms#161] [Q&A]
Hello everyone, I made a satisfaction form on Quillform (studio) today and exported it. When I tried to import it into the Wordpress version of Quillform (premium) that I purchased, this error message appeared. 

Two blocks don't seem to be "known", even though they were present and working correctly on the web version.

How can I fix this? 

![Capture d'écran 2024-01-25 174558](https://github.com/quillforms/quillforms/assets/157730782/a123b68c-c973-46f6-aa92-9dd2ede5a74b)

**Comment by mdmag** [comment]
Hello @rsmpls,

You have to purchase the basic plan of Quill Forms WordPress version since these blocks are available in the basic plan.
Please note that the pricing structure for the studio version is different than the WordPress version.

Best,
Mohamed

**Reply by rsmpls** [reply]
Thanks for your response ! 

But I've already purchased the basic version and i already have put the licence key into position but it still don't work. What should I do then ?

**Comment by mdmag** [comment]
You have to install these addons. Please go to Quill Forms dashboard and click on "Addons" and install "Opinion Scale" Addon and "Custom Thank You Screen".
